docs/targets/logz.md

```diff
@@ -0,0 +1,47 @@
+# Event Target for Logz.io 
+This event target integrates with Logz.io, using received CloudEvent's to ship messages to [Logz.io](https://logz.io/)
+
+## Prerequisites
+* Logz.io account.
+* A Logz.io Shipping [token](https://docs.logz.io/user-guide/tokens/)
+
+
+## Deploying an Instance of the Target
+Open the Bridge creation screen and add a Target of type `Logz`.
+
+![Adding a Logz Target](../images/logz-target/create-bridge-1.png)
+
+In the Target creation form, provide a name for the event Target, and add the following information:
+
+* **Shipping Token** The API Shipping Token. 
+* **Logs Listener URL** The API endpoint. (Can be found above your Shipping token in the Logz.io dashboard)
```

Comment:
Here I would rephrase this as "An API endpoint that can be found above your shipping token in the Logz.io dashboard. Also, shipping does not need to be capitalized here.